### PR TITLE
SEO-172168-Angular-Dialog-control-missing-H1

### PR DESCRIPTION
**Pull request by Omanyadickson**
Hi. Girija-Rajendran ,

Kindly review.
Modifying the missing H1 in PR below.

Regards,
Dickson J

### CODE REVIEW

File: angular/Dialog/how-to.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: how-to
-description: How to
+title: How To Section in Angular Dialog contral | Syncfusion
```
Comment (Sharly-Atieno) [inline angular/Dialog/how-to.md:3]: Control | Syncfusion

Comment (Omanyadickson) [reply angular/Dialog/how-to.md]: renamed

**Review by MallikaRK — COMMENTED** [review]
Approved

File: angular/Dialog/how-to.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: how-to
-description: How to
+title: How To Section in Angular Dialog Control | Syncfusion
```
Comment (Girija-Rajendran) [inline angular/Dialog/how-to.md:3]: can you optimize title and heading?

**Pull request merged**